facebook/delphiFacebook/R/aggregate.R

```diff
@@ -179,6 +179,15 @@ summarize_indicators_day <- function(day_df, indicators, target_day, geo_level,
       var_weight <- indicators$var_weight[row]
       compute_fn <- indicators$compute_fn[[row]]
 
+      # Prevent smoothed weighted signals from being reported for dates after
+      # the latest available weight data.
+      if (target_day > params$latest_weight_date &&
+          indicators$smooth_days[row] == 6 &&
+          indicators$var_weight[row] == "weight") {
```

Comment:
Hmm, let me try to understand these conditions. If `indicators$smooth_days[row] == 1` and weights are unavailable for that day, there will be no observations with non-NA weight, and hence no data reported for that day. That's fine and good. If `indicators$smooth_days[row] > 1`, we could have ≥1 days of data without weights but still have at least some observations with weights, allowing us to report data.

It seems like more robust conditions would hence be

```suggestion
      if (target_day > params$latest_weight_date &&
          indicators$smooth_days[row] > 1 &&
          indicators$var_weight[row] != "weight_unif") {
```

to account for other types of smoothing and other names for weights. Right?